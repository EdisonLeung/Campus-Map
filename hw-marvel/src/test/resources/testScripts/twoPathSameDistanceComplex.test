# tests pathfinding where a node has 2 paths (with multiple jumps) with the
# same distance

# Create Graph
CreateGraph twoPaths

# Add Nodes
AddNode twoPaths a
AddNode twoPaths b
AddNode twoPaths c
AddNode twoPaths d

# Add Edges
AddEdge twoPaths a b AB
AddEdge twoPaths b c BC
AddEdge twoPaths a d AD
AddEdge twoPaths d c DC

# test pathfinding from a to c
FindPath twoPaths a c