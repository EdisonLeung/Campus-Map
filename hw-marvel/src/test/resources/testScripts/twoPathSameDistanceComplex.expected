# tests pathfinding where a node has 2 paths (with multiple jumps) with the
# same distance

# Create Graph
created graph twoPaths

# Add Nodes
added node a to twoPaths
added node b to twoPaths
added node c to twoPaths
added node d to twoPaths

# Add Edges
added edge AB from a to b in twoPaths
added edge BC from b to c in twoPaths
added edge AD from a to d in twoPaths
added edge DC from d to c in twoPaths

# test pathfinding from a to c
path from a to c:
a to b via AB
b to c via BC